modules/deleting-the-cluster-logging-operator-by-using-the-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+// * uninstalling/uninstalling-logging.adoc
+
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-07-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-the-cluster-logging-operator-by-using-the-web-console_{context}"]
+= Deleting the {clo} by using the web console
+
+Delete the {clo} by using the {product-title} web console.
+
+.Prerequisites
+* You have administrator permissions.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{clo}* from the *Installed Operators* list and go to the *Cluster Log Forwarder* tab.
+
+. Click {kebab} for the `ClusterLogForwarder` entry and select *Delete ClusterLogForwarder*.
+
+. Click the *Actions* button and click *Uninstall Operator*.
```

Comment:
This can be simpler, please replicate this for Loki Operator
```suggestion
. Find *{clo}* in the list and click {kebab} then press *Uninstall Operator*.

. In the pop-up select *Delete all operand instances for this operator* and then press *Uninstall*
```